src/chat/chatSelectors.js

```diff
@@ -13,3 +16,16 @@ export const getPointer = (state) => {
 
   return [messages[0].id, messages[messages.length - 1].id];
 };
+
+export const getRecentConversations = (state) => {
+  const selfEmail = state.account[0].email;
+  const messages = state.chat.messages[JSON.stringify(specialNarrow('private'))] || [];
+  const recipientSet = messages.reduce((accumulator, x) => {
```

Comment:
This is hard to read as a reduce pattern. Reduce usually implies that N elements get condensed to 1 element.

I think it's much easier to read if made imperative:
```
const recipients = new Set();
for (const message of messages) {
  ...
  recipients.add(normalized);
}
return recipients;
```

Alternatively:
```
return unique(
  messages
  .map((msg) => normalizeRecipients(msg.display_recipient))
  .filter((r) => r.email !== selfEmail)
);
```
where
```
unique = (list) => [...new Set(list)];
```